docroot/themes/custom/mass_theme/templates/paragraphs/paragraph--org-section-long-form--default.html.twig

```diff
@@ -1,14 +1,17 @@
-{% embed "@organisms/by-author/rich-text.twig" with {
-  "richText": {
-    "compHeading": {
-      "title": (content.field_section_long_form_heading|field_value)
-    }
-  }
+{% embed "@organisms/by-author/stacked-row-section.twig" with {
```

Comment:
Glad you figured this out. It fixed all of the front-end issues I was seeing. ++

Reply:
Thank you 😄